### PR TITLE
Fix akri-app.

### PR DESCRIPTION
**Pull request by ensonic**
* Add subdir for robot deploymnet. Without it we never pull the workloads.
* Remove curly braces from the akri-crd. These cause errors when applying this as part of a chart.
* Simply the update script. We'll do customization via values.yaml

### CODE REVIEW

File: third_party/akri/update-akri.sh [hunk 1]
```diff
@@ -3,48 +3,9 @@
 # fetch the latest version
 helm repo add akri-helm-charts https://project-akri.github.io/akri/ 
 helm repo update akri-helm-charts
-helm pull akri-helm-charts/akri --untar
+helm pull akri-helm-charts/akri
```
Comment (csieber) [inline third_party/akri/update-akri.sh:6]: would it make sense to pin a version here?

File: third_party/akri/update-akri.sh [hunk 2]
```diff
@@ -3,48 +3,9 @@
...
-# opcua - May require more updates
-sed -i 's/.Chart.AppVersion/.Values.akriAppVersion/g' ./akri/templates/opcua-discovery-handler.yaml
-
-# Repackage updated helm-charts 
-helm package ./akri
-
-# Cleanup
-rm -rf ./akri
-
+tar xzf akri-*.tgz --strip-components=2 akri/crds/
```
Comment (csieber) [inline third_party/akri/update-akri.sh:9]: if the old tgz is not deleted, wouldn't this result in the old and new version both being extracted?

File: third_party/akri/update-akri.sh (hunk 1 above)
Comment (ensonic) [reply third_party/akri/update-akri.sh:6]: Good idea. Done.

File: third_party/akri/update-akri.sh (hunk 2 above)
Comment (ensonic) [reply third_party/akri/update-akri.sh:9]: Using the version here now.

**Pull request merged**